cmd/kubeadm/app/apis/kubeadm/validation/validation.go

```diff
@@ -381,13 +381,25 @@ func ValidateIPNetFromString(subnet string, minAddrs int64, fldPath *field.Path)
 	return allErrs
 }
 
+// ValidatePodSubnetsFromString validates network portion of pod subnet(s)
+func ValidatePodSubnetsFromString(cidrStr string, fldPath *field.Path) field.ErrorList {
```

Comment:
you should call only this function if dual stack is enabled, you also have to check that there are 2 different families , I think that something like this https://github.com/kubernetes/kubernetes/blob/54851658f3706019639c258ea0a4975e8737f2e8/test/e2e/network/dual_stack.go#L257-L260

Reply:
my bad, sorry, reading the previous comment I see that kubeadm doesn´t validate on feature gates 😓